packages/styled-base/src/utils.js

```diff
@@ -1,9 +1,32 @@
 // @flow
 import * as React from 'react'
+import type { ElementType } from 'react'
 import isPropValid from '@emotion/is-prop-valid'
 
 export type Interpolations = Array<any>
 
+export type StyledOptions = {
+  label?: string,
+  shouldForwardProp?: string => boolean,
+  target?: string
+}
+
+export type StyledComponent<P> = React$StatelessFunctionalComponent<P> & {
```

Comment:
interesting, did u manage finally to extend this builtin type? 🤔 Also - shouldn't we refer to this as `React.StatelessFunctionalComponent`?

Reply:
Yeah I don't know what happened, it works now 😂

I can change it, but just FYI, internally, `React.StatelessFunctionalComponent` aliases to `React$StatelessFunctionalComponent`

Reply:
Yeah - I know that, but `React$StatelessFunctionalComponent` looks like accessing something private :p